src/main/java/org/sqlite/hibernate/dialect/SQLiteDialect.java

```diff
@@ -327,9 +331,46 @@ public UniqueDelegate getUniqueDelegate() {
 		private SQLiteUniqueDelegate(Dialect dialect) {
 			super( dialect );
 		}
+
+		/**
+		 * SQLite use table creation sql to define unique constraints.
+		 */
 		@Override
 		public String getColumnDefinitionUniquenessFragment(Column column) {
-			return " unique";
+			return "";
+		}
```

Comment:
And you can remove this override: see https://github.com/hibernate/hibernate-orm/blob/master/hibernate-core/src/main/java/org/hibernate/dialect/unique/DefaultUniqueDelegate.java#L37-L39

Reply:
Is it better to leave the overridden method here? Because this is more explicit, means we have processed this situation and it return an empty string. What you think?

Reply:
Fine.
Thanks.